docs/workflow.rst

```diff
@@ -238,12 +233,13 @@ Let's get in more detais:
 
```

Comment:
upstream (remote from official repo) => from the official repository.

Make sure we use `repository` instead of `repo` though out the docs.